inst/shiny-server/global.R

```diff
@@ -56,19 +56,19 @@ plotlyOutput.IOHanalyzer <- function(outputId, width = '100%', aspect_ratio = 16
   padding_bottom <- paste0(100 / aspect_ratio, '%')
   reportSize <- TRUE
   inline <- FALSE
-  
+
   checkShinyVersion()
   html <- htmltools::tagList(
-    widget_html('plotly', 'plotly', id = outputId, 
-                class = paste0('plotly', " html-widget html-widget-output", 
-                               if (reportSize) 
-                                 " shiny-report-size"), 
-                style = sprintf("width:%s; height: 0; padding-bottom:%s; %s", 
-                                htmltools::validateCssUnit(width), 
-                                htmltools::validateCssUnit(padding_bottom), 
-                                if (inline) 
+    widget_html('plotly', 'plotly', id = outputId,
+                class = paste0('plotly', " html-widget html-widget-output",
+                               if (reportSize)
+                                 " shiny-report-size"),
+                style = sprintf("width:%s; height: 0; padding-bottom:%s; %s",
+                                htmltools::validateCssUnit(width),
```

Comment:
This is only to check the format of CSS contents?

Reply:
This wasn't changed, just marked as such because of an incorrect setting on my local git (line-ending difference between linux and windows was not turned off)